### PR TITLE
OHRM5X-1106: Project Report API linked with FE

### PR DESCRIPTION
**Pull request by VageeshOrangeHRM**
This PR Contains following changes:
1. Project Report API Connected with the FE

### CODE REVIEW

File: symfony/plugins/orangehrmTimePlugin/Controller/ProjectActivityReportController.php [hunk 1]
```diff
@@ -40,13 +43,17 @@ public function preRender(Request $request): void
 
         if ($request->query->has('projectId')) {
             // TODO: Get project object
```
Comment (Super-Chama) [inline symfony/plugins/orangehrmTimePlugin/Controller/ProjectActivityReportController.php:45]: Remove this comment

**Review by Super-Chama — APPROVED** [review]
Hope you will update the employee report as well ?

File: symfony/plugins/orangehrmTimePlugin/Controller/ProjectActivityReportController.php (hunk 1 above)
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmTimePlugin/Controller/ProjectActivityReportController.php]: done

**Review by RajithaKumara — CHANGES_REQUESTED** [review]
Can you remove mock controllers and routes

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**